services/server/config/config.go

```diff
@@ -203,6 +213,24 @@ func (bc *BoltConfig) Validate() error {
 	}
 }
 
+const (
+	// ExporterTypeOTLP represents the open telemetry exporter OTLP
+	ExporterTypeOTLP = "otlp"
+)
+
+// Validate openTelemetry config
+func (cfg *OpenTelemetryConfig) Validate() error {
+	switch cfg.ExporterName {
+	case ExporterTypeOTLP:
+		if cfg.ServiceNameKey == "" {
+			return errors.Wrapf(errdefs.ErrInvalidArgument, "missing service name in config %v", cfg)
+		}
+		return nil
+	default:
+		return errors.Wrapf(errdefs.ErrInvalidArgument, "unsupported exporter: %v", cfg)
```

Comment:
It may be better to use a specific field or `%+v` format.

https://pkg.go.dev/fmt

> when printing structs, the plus flag (%+v) adds field names

Reply:
Changed to %+v